frontend/src/router.js

```diff
@@ -228,9 +228,16 @@ export default new Router({
           component: () =>
             import(/* webpackChunkName: "campAdmin" */ './views/camp/Admin.vue'),
         },
+        {
+          path: 'program',
+          name: 'camp/program',
+          async beforeEnter(to, from, next) {
+            redirectToPeriod(to, from, next, 'camp/program/period')
+          },
+        },
```

Comment:
The order of these routes matters. Did you have an explicit reason to move this up?

Reply:
[db6bb6c](https://github.com/ecamp/ecamp3/pull/3038/commits/db6bb6ce2db90be0ce9a124f207ce9751d8dfc8c), I checked the routing behavior and it looks ok to me